src/amo/components/AddonReview.js

```diff
@@ -104,15 +94,6 @@ export class AddonReviewBase extends React.Component {
         <form onSubmit={this.onSubmit} ref={(ref) => { this.reviewForm = ref; }}>
           <div className="AddonReview-form-input">
             {errorHandler.hasError() ? errorHandler.renderError() : null}
-            <input
```

Comment:
Yeah, a hidden one will give extra context to screenreaders. Something simple like "Your review text" should be fine.